### PR TITLE
Replace translation and calendar image icons

### PR DESCRIPTION
**Pull request by agh1**
Overview
----------------------------------------
This continues the ongoing work to use font icons rather than images--now changing the way that translation and date widgets work.

Before
----------------------------------------
The translation icon is two squares, which is meaningless.  The calendar is an old-timey GIF.
screenshot_2018-10-24 title and settings help support civicrm creepyeyeball wordpress 1
screenshot_2018-10-24 title and settings help support civicrm creepyeyeball wordpress 3

After
----------------------------------------
The translation icon is [`fa-language`](https://fontawesome.com/v4.7.0/icon/language).  The date widget now uses a placeholder [`fa-calendar'](https://fontawesome.com/v4.7.0/icon/calendar)
screenshot_2018-10-24 title and settings help support civicrm creepyeyeball wordpress
screenshot_2018-10-24 title and settings help support civicrm creepyeyeball wordpress 2

Technical Details
----------------------------------------
Placeholder text disappears when there's a value, so note that in the screenshot of the date widget.  Even though it's not necessarily an ideal change in some folks' opinion, it is one towards standardization.

### CODE REVIEW

**Review by aydun — COMMENTED** [review]
Another small step for mankind ...  Looks good.
Don't want to be too picky but can we use something symbolic or more intuitive to the coder than '\uF073'?  If not, just a comment would help future readers.

**Comment by agh1** [comment]
@aydun I don't think there's any way around the `\uF073` aside from adding the &#xF073; character to the code (which would be equally obtuse without the font, and potentially mangled in certain environments). 
 But it's a good suggestion to make it clear for developers--I just added a comment.

File: css/civicrm.css [hunk 1]
```diff
@@ -2683,12 +2683,15 @@ div.crm-container form {
...
-  background: white url(../i/cal.gif) no-repeat scroll right center;
-  padding-right: 16px;
-  /* so that text doesn't flow on top of icon */
   width: 9em;
 }
 
+.crm-container input.dateplugin::placeholder,
+.crm-container input.crm-form-date::placeholder {
+  font-family: "FontAwesome";
+  text-align: right;
```
Comment (colemanw) [inline css/civicrm.css]: I believe you can also set the "content" attribute here so it's not needed with js.
(I always prefer css first if possible before resorting to js)

Comment (agh1) [reply css/civicrm.css]: That was my first instinct, too, but it doesn't work.  Or, rather when I tried it, it doesn't work, and apparently [it's webkit only](https://stackoverflow.com/questions/8075986/how-to-set-placeholder-value-using-css) and I was in Firefox.

Comment (colemanw) [reply css/civicrm.css]: Ok, how about we do both for now, with a comment in the js that it's a FF shim that can be removed someday.

**Comment by JoeMurray** [comment]
Maybe it's just my eyes or the resolution of the posted image from @agh1 but I'm unable to see the left part of the fa-language as a Chinese character. The existing icon seems to be larger in the image compared to the textbox than the new fa-language one. Could we make it very slightly bigger to help make it more comprehensible?? (Extra ? because I'm not sure this is a workable/good suggestion.)

File: css/civicrm.css (hunk 1 above)
Comment (agh1) [reply css/civicrm.css]: Oops I should've read the comments over on SE--it doesn't work in Webkit anymore either.  JS it is.

**Comment by agh1** [comment]
@JoeMurray it is a very busy icon (one of the few in FA that have letters within them).  I did use the `fa-lg` class to make it bigger than icons would normally be in that font size, and that seems to make the icon's total size approximately the same as the old one.  I'm a little hesitant to go tinkering further with the size since that would involve CSS that's specific to that icon.

**Comment by JoeMurray** [comment]
> @JoeMurray it is a very busy icon (one of the few in FA that have letters within them). I did use the `fa-lg` class to make it bigger than icons would normally be in that font size, and that seems to make the icon's total size approximately the same as the old one. I'm a little hesitant to go tinkering further with the size since that would involve CSS that's specific to that icon.

I agree on not using css. I take it that fa-2x looks too big?

**Comment by agh1** [comment]
@JoeMurray it seems to dwarf the labels: 
image

**Comment by colemanw** [comment]
Merging to prevent further bikeshedding ;)

**Pull request merged**